schema/Makefile

```diff
@@ -5,10 +5,13 @@
 YAMLS:=vrs.yaml
 JSONS:=${YAMLS:.yaml=.json}
 
-all: ${JSONS} defs
+all: vrs.json defs
```

Comment:
What about ga4gh.json?

Reply:
AFAIK, ga4gh.json isn't used anywhere, and unlike vrs.json, isn't JSON Schema.

It's not a big deal to put it back in, but is it actually our intent to provide this file for some purpose, or is the YAML sufficient?

Reply:
I provided for completeness (json and yaml pairs), but your logic is sound and that file should probably move to ga4gh core anyway. So, let's keep as-proposed and we'll revive if needed.